tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -9327,7 +9345,7 @@ TEST_F(EnergyPlusFixture, UnitarySystem_GetInputATMixerInlet)
         "AirLoopHVAC:UnitarySystem,",
         "  East Zone Unitary System,       !- Name",
         "  Setpoint,                       !- Control Type",
-        "  East Zone,                      !- Controlling Zone or Thermostat Location",
+        "  ,                               !- Controlling Zone or Thermostat Location",
```

Comment:
Removed the control zone name here and 2 places below since control zone name is not needed for Control Type = Setpoint.